onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -1300,30 +1301,30 @@ def output_latency(results, csv_filename):
 
     with open(csv_filename, mode="a", newline='') as csv_file:
         column_names = ["Model",
-                        "CPU \nmean (ms)",
-                        "CPU \n 90th percentile (ms)",
+                        "CPU fp32 \nmean (ms)",
```

Comment:
I wonder if it's possible to make the column names more maintainable.
I see some here, and some in other scripts (memory etc.)
and it's hard to debug whether data is going under the right place if you make a change later.
think about how we can improve this?

Reply:
the new dashboard used Azure Tables - looking into how to integrate that.

Reply:
Created a new ticket for this https://msdata.visualstudio.com/Vienna/_boards/board/t/ONNX%20Inference/Features/?workitem=1151323.